src/pages/Bridge/index.tsx

```diff
@@ -132,32 +184,71 @@ const BridgePage: React.FC = () => {
     const formValues = getValues();
 
     handleChainFieldChange({
-      fromChainId: formValues.toChainId,
-      toChainId: formValues.fromChainId,
+      newFromChainId: formValues.toChainId,
+      newToChainId: formValues.fromChainId,
     });
   }, [getValues, handleChainFieldChange]);
-
-  const onSubmit = (values: z.infer<typeof formSchema>) => {
-    // TODO: send transaction (see VEN-2248)
-    console.log(values);
+  const { mutateAsync: bridgeXvs } = useBridgeXvs();
+
+  const onSubmit = async () => {
+    if (accountAddress) {
+      try {
+        await bridgeXvs({
+          accountAddress,
+          amountMantissa,
+          destinationChainId: toChainId,
+          nativeCurrencyFeeMantissa: bridgeEstimatedFeeMantissa,
+        });
```

Comment:
I think we should reset the form when the transaction is successfully sent, to be consistent with the other forms' behavior.